app/views/vacancies/edit.html.erb

```diff
@@ -0,0 +1,35 @@
+<h2 class="page-title"><%= t(".edit_vacancies") %></h2>
+
+<div class="panel panel-default">
```

Comment:
you need to extract the entire form to a partial `_form.html.erb`
and use it on `edit.html.erb` and `new.html.erb`